newclient/src/modules/blocks/handlers/BlockStats/BlockStats.vue

```diff
@@ -97,7 +97,7 @@ export default class BlockStats extends Mixins(NumberFormatMixin) {
         if (this.loading) {
             return ''
         }
-        if (this.newBlock) {
+        if (this.newBlock && new BN(this.newBlock).isGreaterThan(new BN(this.initialLoadedBlock))) {
```

Comment:
This would only make sure that it is higher then initialBlock Loaded, and will pass for:
initialBlock = x
1st blockUpdate = x+1
2nd blockUpdate = x+2
**3rd blockUpdate = X+1**
it will display 3rd blockUpdate, when it should stays at the 2nd blockUpdate